### PR TITLE
adding badges and nightly links

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+
 This repository contains a work-in-progress eventing components using RabbitMQ
 as an implementation.
 
 > See <https://github.com/knative/eventing-contrib/pull/336> for background /
 > discussion.
 
+## Nightly Artifacts
+
+For RabbitMQ Broker,
```
Comment (mattmoor) [inline README.md:19]: Format markdown:
```suggestion
For RabbitMQ Broker,
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
 
+## Nightly Artifacts
+
+For RabbitMQ Broker, 
+
+```shell script
+kubectl apply -f https://storage.googleapis.com/knative-nightly/eventing-rabbitmq/latest/rabbitmq-broker.yaml
+```
+
+For RabbitMQ Source,
```
Comment (mattmoor) [inline README.md:25]: Format markdown:
```suggestion
For RabbitMQ Source,
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Comment by vaikas** [comment]
Sweet!
/lgtm
/approve

**Pull request merged**